devops/Makefile

```diff
@@ -490,127 +489,72 @@ deploy-forms:
 	$(call rollout_and_wait,dc/$(FORMS_NAME))
 
 # Create database backup structure (build/configmap/deploy)
-
-# Create the config map to be shared by POSTGRES and mongodb.
-db-backup-configmap:
-	test -n "$(NAMESPACE)"
-	test -n "$(BACKUP_POSTGRESQL_APP_NAME)"
-	test -n "$(BACKUP_MONGODB_APP_NAME)"
-	@echo "+\n++ Creating configmap resources for database backups on $(NAMESPACE)\n+"
-	oc -n $(NAMESPACE) create configmap $(BACKUP_CONFIGMAP_NAME) --from-file=./openshift/database-backup/backup.conf
-
-db-backup-configmap-delete:
-	test -n "$(NAMESPACE)"
-	@echo "+\n++ Removing configmap resoures for database backups on $(NAMESPACE)\n+"
-	oc -n $(NAMESPACE) delete configmap/$(BACKUP_CONFIGMAP_NAME)
-
 # Create the POSTGRES structure
-db-backup-build-postgresql:
+db-backup-build-patroni:
```

Comment:
Should this also include SIMS to be aligned with the other renamed variables?

Reply:
i specifically did not add SIMS, becuase build is going to be same for all, but while deploying we can have multiple databases with different names.